tests/test_server.py

```
@@ -0,0 +1,417 @@
+import asyncio
+import time
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+from proxybroker import Proxy
+from proxybroker.errors import NoProxyError, ProxyConnError, ProxyTimeoutError
```

Ruff (0.11.9)

`proxybroker.errors.NoProxyError` imported but unused

Remove unused import

(F401)